ezGroceriesShoppingList/src/main/java/com/ezgroceries/shoppinglist/service/CocktailService.java

```diff
@@ -0,0 +1,12 @@
+package com.ezgroceries.shoppinglist.service;
+
+import com.ezgroceries.shoppinglist.entity.Cocktail;
+
+import java.util.List;
+
+public interface CocktailService {
+
+    List<Cocktail> listCocktails(String search);
```

Comment:
Following naming convention either use the 'list' word inside the name of the function to indicate that it is not a single record OR use the plural form of words e.x.(cocktailS). But don't use both at the same time  
Imo  a better naming for this function will be: 
   -> getCoctails (I prefer this one) 
   ->getCoctailList